hail/src/main/scala/is/hail/services/package.scala

```diff
@@ -83,6 +83,28 @@ package object services {
       case e: HttpResponseException
           if RETRYABLE_HTTP_STATUS_CODES.contains(e.getStatusCode()) =>
         true
+      case e: HttpResponseException
+          if (e.getStatusCode() == 401 &&
```

Comment:
Is this supposed to be 410 to match the output below?

Reply:
Yep, good catch!